src/eve-server/ship/ShipService.cpp

```diff
@@ -1001,6 +1004,7 @@ PyResult ShipBound::Jettison(PyCallArgs &call, PyList* itemIDs) {
                 throw CustomError ("Unable to spawn jetcan.");
             // create new container
             ContainerSE* cSE = new ContainerSE(jcRef, this->GetServiceManager(), pSysMgr, data);
+            cSE->Init();
```

Comment:
Since the ContainerSE was never initialized, all of its values where 0/null/etc... this was then sent to the client on the next destiny update causing whatever grid this container was in to become corrupted.  At least until the system was unloaded or the server was otherwise rebooted.

Reply:
There's a few other SystemEntity items that get created in this method, but I'm unsure if they should have their Init()s called on them as well.